components/service/glean/src/test/java/mozilla/components/service/glean/ping/BaselinePingTest.kt

```diff
@@ -47,4 +50,42 @@ class BaselinePingTest {
             )
         )
     }
+
+    @Test
+    fun `getLanguageTag() reports the tag for the default locale`() {
+        val baselinePing = BaselinePing(ApplicationProvider.getApplicationContext<Context>())
+        val defaultLanguageTag = baselinePing.getLanguageTag()
+
+        assertNotNull(defaultLanguageTag)
+        assertFalse(defaultLanguageTag.isEmpty())
+        assertEquals("en-US", defaultLanguageTag)
```

Comment:
Will this fail if the testing machine's locale is set to something non-English?  Should we instead mock `Locale.getDefault()`?

Reply:
Good point. I cross checked this, and turns out that we can't have anything other than "en-US" since we're runnign the tests in Robolectric. [See here](http://robolectric.org/javadoc/4.0/org/robolectric/shadows/ShadowLocaleData.html).